### ISSUE TITLE
我用一个有透明边框的webp动画测试，发现透明边框显示成了白色，能帮忙看下这个问题吗

### ISSUE THREAD

**Issue by WuGang-128** [penfeizhou/APNG4Android#45]
### New Issue Checklist

- [ ] I have searched for a similar issue in the [project](https://github.com/penfeizhou/APNG4Android/issues) and found none

### Issue Info

| Info                         | Value                                                 |      |
| ---------------------------- | ----------------------------------------------------- | ---- |
| Device Info                  | e.g. Huawei Mate 30 pro                               |      |
| System Version               | e.g.  6.0                                             |      |
| APNG4Android Library Version | e.g.  1.0.0                                           |      |
| Repro rate                   | e.g. all the time (100%) / sometimes x% / only once   |      |
| Repro with our demo project  | e.g. does it happen with our demo project?            |      |
| Demo project link            | e.g. link to a demo project that highlights the issue |      |

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by penfeizhou** [comment]
WebP支持动图，请看下[https://isparta.github.io/compare-webp/image/gif_webp/webp/2.webp](url)这个动图，就是透明底色的。

**Comment by WuGang-128** [comment]
demo里  word-cup.webp  这个动图显示就有问题，透明区域显示成白色了 gang wu 邮箱：wgnobg@gmail.com 签名由 网易邮箱大师 定制 在2019年11月07日 09:34，osborn 写道： WebP支持动图，请看下https://isparta.github.io/compare-webp/image/gif_webp/webp/2.webp这个动图，就是透明底色的。 — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by penfeizhou** [comment]
![image](https://user-images.githubusercontent.com/9526211/68354423-4c24b000-0147-11ea-883c-cd9c01652e0a.png)
是两个角吗？这个区域确实是白色，因为设置了白色作为动画背景色了

**Comment by penfeizhou** [comment]
`Background Color: 32 bits (uint32)
The default background color of the canvas in [Blue, Green, Red, Alpha] byte order. This color MAY be used to fill the unused space on the canvas around the frames, as well as the transparent pixels of the first frame. Background color is also used when disposal method is 1.
Note:

Background color MAY contain a transparency value (alpha), even if the Alpha flag in VP8X chunk is unset.

Viewer applications SHOULD treat the background color value as a hint, and are not required to use it.

The canvas is cleared at the start of each loop. The background color MAY be used to achieve this.`

**Comment by WuGang-128** [comment]
在魅族M5  note上显示有问题 gang wu 邮箱：wgnobg@gmail.com 签名由 网易邮箱大师 定制 在2019年11月07日 10:08，gang wu 写道： demo里  word-cup.webp  这个动图显示就有问题，透明区域显示成白色了 gang wu 邮箱：wgnobg@gmail.com 签名由 网易邮箱大师 定制 在2019年11月07日 09:34，osborn 写道： WebP支持动图，请看下https://isparta.github.io/compare-webp/image/gif_webp/webp/2.webp这个动图，就是透明底色的。 — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by penfeizhou** [comment]
http://a.webpurr.com/Rqa.webp
这个图片背景色被定义为纯白，但在Chrome上背景色是透明色。
我们需要check一下chrome的解码逻辑。

**Comment by WuGang-128** [comment]
是中间球运动位置 gang wu 邮箱：wgnobg@gmail.com 签名由 网易邮箱大师 定制 在2019年11月07日 11:05，osborn 写道： http://a.webpurr.com/Rqa.webp 这个图片背景色被定义为纯白，但在Chrome上背景色是透明色。 我们需要check一下chrome的解码逻辑。 — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by penfeizhou** [comment]
你好，可升级到1.2.0版本，修复了改问题
感谢反馈。
